toolkits/confluence/arcade_confluence/tools/search.py

```diff
@@ -0,0 +1,51 @@
+from typing import Annotated
+
+from arcade.sdk import ToolContext, tool
+from arcade.sdk.auth import Atlassian
+from arcade.sdk.errors import ToolExecutionError
+
+from arcade_confluence.client import ConfluenceClientV1
+
+
+@tool(
+    requires_auth=Atlassian(
+        scopes=["search:confluence"],
+    )
+)
+async def search_content(
+    context: ToolContext,
+    terms: Annotated[
+        list[str] | None,
+        "A list of single words to search for in content. "
+        "For example, ['project', 'documentation']",
+    ] = None,
+    phrases: Annotated[
+        list[str] | None,
+        "A list of groups of words that match content containing all the words in any order. "
+        "For example, ['software developer']",
+    ] = None,
+    enable_fuzzy: Annotated[
+        bool,
+        "Enable fuzzy matching to find similar terms (e.g. 'roam' will find 'foam'). "
+        "Defaults to False",
+    ] = False,
```

Comment:
In general, I think people expect search to be fuzzy by default. It's the experience of most modern search engines and search features in SaaS GUIs